src/visuals/olympic-charts/OlympicsChart.js

```diff
@@ -0,0 +1,1835 @@
+      });
+    }
+  }
+
+  viewSport(d) {
+    this.setState({
+      viewSport: d.data.title,
+    });
+  }
+
+  // Render Method
+  render() {
+    const {
+      left, right, refAreaLeft, refAreaRight,
+    } = this.state;
+
+    const timeFrameData = allData.filter((d) => d['Olympic Year'] >= left && d['Olympic Year'] <= right);
+
+    // Aggregate per year
+    const byYear = {};
+    timeFrameData.forEach((d) => {
+      if ((this.state.sport === null || d.Sport === this.state.sport)) {
+        if (byYear[d['Olympic Year']] === undefined) {
+          byYear[d['Olympic Year']] = {
+            Gold: 0,
+            Silver: 0,
+            Bronze: 0,
+            Medals: 0,
+          };
+        }
+        byYear[d['Olympic Year']].Gold += d.Gold;
+        byYear[d['Olympic Year']].Silver += d.Silver;
+        byYear[d['Olympic Year']].Bronze += d.Bronze;
+        byYear[d['Olympic Year']].Medals += d.Gold + d.Silver + d.Bronze;
+      }
+    });
+    const yearData = [];
+    Object.keys(byYear).forEach((d) => {
+      yearData.push({
+        Year: d,
+        Gold: byYear[d].Gold,
+        Silver: byYear[d].Silver,
+        Bronze: byYear[d].Bronze,
+        Medals: byYear[d].Medals,
+      });
+    });
+
+    // Aggregate across time
+    const byAthlete = {};
+    timeFrameData.forEach((d) => {
+      if (byAthlete[d.Name] === undefined) {
+        byAthlete[d.Name] = {
+          Sport: d.Sport,
+          Gold: 0,
+          Silver: 0,
+          Bronze: 0,
+        };
+      }
+      byAthlete[d.Name].Gold += d.Gold;
+      byAthlete[d.Name].Silver += d.Silver;
+      byAthlete[d.Name].Bronze += d.Bronze;
+    });
+
+    const athleteData = [];
+    Object.keys(byAthlete).forEach((d) => {
+      athleteData.push({
+        Name: d,
+        Sport: byAthlete[d].Sport,
+        Gold: byAthlete[d].Gold,
+        Silver: byAthlete[d].Silver,
+        Bronze: byAthlete[d].Bronze,
+      });
+    });
+
+    // Treemap Data
+    const bySport = {};
+    athleteData.forEach((d) => {
+      if (this.state.sport === null || d.Sport === this.state.sport) {
+        if (bySport[d.Sport] === undefined) {
+          bySport[d.Sport] = 0;
+        }
+        bySport[d.Sport] += d.Gold + d.Silver + d.Bronze;
+      }
+    });
+    const treeData = [];
+    Object.keys(bySport).forEach((d) => treeData.push({
+      title: d,
+      size: bySport[d],
+      style: {
+        border: '1px solid white',
+      },
+    }));
+
+    const targetSport = this.state.viewSport === null ? this.state.sport : this.state.viewSport;
+
+    const filteredData = athleteData
+      .filter((d) => {
+        if (targetSport === null) return true;
+        return d.Sport === targetSport;
+      })
+      .sort((a, b) => (
+        3 * b.Gold
+          - 3 * a.Gold
+          + 2 * b.Silver
+          - 2 * a.Silver
+          + b.Bronze
+          - a.Bronze
+      ))
+      .filter((d, i) => i < 10);
+
+    // Sizes
+    const height = window.innerHeight - 150;
```

Comment:
We cannot call window in Gatsby! Read [here](https://www.gatsbyjs.com/docs/how-to/local-development/troubleshooting-common-errors/#error-referenceerror-window-is-not-defined-when-running-gatsby-build). 
Also, I'm thinking it's not absolutely necessary to call window here - we could just hard code (oof) ? Lmk what you think tho